test/cli-test.js

```diff
@@ -7,13 +7,31 @@ var cliTxt = path.resolve(_files, 'cli.txt');
 var cliPartialsTxt = path.resolve(_files, 'cli_with_partials.txt');
 var moduleVersion = require('../package').version;
 
-var exec = require('child_process').exec;
+var child_process = require('child_process');
+
+function changeForOS(command) {
+
+  if(process.platform === 'win32') {
+    return command.
+      replace(/bin\/mustache/g, 'node bin\\mustache').
+      replace(/\bcat\b/g, 'type').
+      replace(/\//g, '\\');
```

Comment:
Personally, I prefer having dots at the end of a line in these cases because 1) I'm paranoid about JS's semi-colon insertion (even though I've never had a semi-colon inserted where I didn't want one), and 2) just seeing `return command` sort of implies that that is all you want to return, having the dot at the end implies that's not the end of the statement.

That aside, I will, of course, make my changes consistent with the rest of the code base :)